equinox-shipping/Domain/Types.fs

```diff
@@ -0,0 +1,12 @@
+module Types
+
+open FSharp.UMX
+
+[<Measure>] type shipmentId
+[<Measure>] type containerId
+[<Measure>] type transactionId
+
+type ShipmentState = { association: string<containerId> option }
```

Comment:
this should not be global - its fine to nest within the `module` of the Aggregate in a local `module Types` but its definitely not useful to have a global place where you put State stuff (e.g. if you had 20 aggregates, this would get very messy and merge conflicty and you'd never be able to spot which are dead - also you want a tripwire when people are adding Yet More Coupling)